standalone/src/models/discount-group/builders.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { DiscountGroupRest, DiscountGroupGraphql } from './index';
+
+describe('DiscountGroup Builder', () => {
+  it('should build properties for the REST representation', () => {
+    const restModel = DiscountGroupRest.random().build();
+
+    expect(restModel).toEqual(
+      expect.objectContaining({
+        id: expect.any(String),
+        createdAt: expect.any(String),
+        createdBy: expect.objectContaining({
+          clientId: expect.any(String),
+          externalUserId: expect.any(String),
+        }),
+        lastModifiedAt: expect.any(String),
+        lastModifiedBy: expect.objectContaining({
+          clientId: expect.any(String),
+          externalUserId: expect.any(String),
+        }),
+        key: expect.any(String),
+        version: expect.any(Number),
+        sortOrder: expect.any(String),
+        name: expect.objectContaining({
+          en: expect.any(String),
+          de: expect.any(String),
+        }),
+        description: expect.objectContaining({
+          en: expect.any(String),
+          de: expect.any(String),
+        }),
+      })
+    );
+  });
+  it('should build properties for the GraphQL representation', () => {
+    const graphqlModel = DiscountGroupGraphql.random().build();
+
+    expect(graphqlModel).toEqual(
+      expect.objectContaining({
+        id: expect.any(String),
+        createdAt: expect.any(String),
+        createdBy: expect.objectContaining({
+          clientId: expect.any(String),
+          externalUserId: expect.any(String),
+        }),
```

Comment:
For nested models in the GraphQL builder it's fine to just check for the `__typename` property. If it says it's the right type, we can trust on that model's tests that the value will be correctly generated.

```suggestion
        createdBy: expect.objectContaining({
          __typename: 'Initiator',
        }),
```

(_we will document this_)